README.md

```diff
@@ -120,10 +120,12 @@ performance.
      type="application/javascript"
    ></script>
    <script type="application/javascript">
-     littlefoot.default()
+     littlefoot.littlefoot()  // Pass any littlefoot settings here.
    </script>
    ```
 
+If you want to use the default options, use `littlefoot.default()` instead.
```

Comment:
I would remove this line, the comment seems to indicate the functions are different when they are not. `littlefoot()` works for all cases and is less confusing.